packages/create-app/templates/default-app/app-config.local.yaml

```diff
@@ -0,0 +1 @@
+# Backstage override configuration for your local development environment
```

Comment:
```suggestion
# Backstage override configuration for your personal local development environment. This file is in .gitignore and therefore should never be checked into source control.
```

maybe worth highlighting, so that it can't be mixed up with what `.development.yaml` would have been

Reply:
love it